monitoring-dashboard/components/org.wso2.micro.integrator.dashboard.web/web-app/src/utils/HTTPClient.js

```diff
@@ -138,6 +138,17 @@ export default class HTTPClient {
         });
     }
 
+    static getCappArtifacts(groupId, nodeId, artifactName) {
+        const resourcePath = `${groupId}/${Constants.PREFIX_NODES}/${nodeId}/capps/${artifactName}/artifacts`
+        return new Promise((resolve, reject) => {
+            this.getResource(resourcePath).then(response => {
+                resolve(response)
+            }).catch(error => {
+                reject(error)
+            })
+        });
```

Comment:
Since `getResource` returns a Promise, we don't need to explicitly return a Promise here. We can simply call,
```suggestion
        return this.getResource(resourcePath)
```

Reply:
Fixed